index.html

```diff
@@ -351,6 +351,55 @@ <h4 class="featureText">Terminal</h4>
           </li>
         </ul>
       </section>
+      <section class="flex-container" id="formContainer">
+        <p id="contactDescription">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/xnqyndrw" method="POST">
+          <ul class="flex-container" id="formElements">
+            <li class="formElement">
+              <input class="fElement" type="text" name="name" placeholder="Full name" maxlength="30" required>
+            </li>
+            <li class="formElement">
+              <input type="email" name="email" placeholder="Email address" required>
+            </li>
+            <li class="formElement">
+              <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+            </li>
```

Comment:
- [ ] As requested by the previous reviewer [here](https://github.com/evillaz/my-portfolio/pull/4#discussion_r1072856120), kindly remove the unordered list syntax to make your project semantically correct:

here is how the code might be :+1: 

```
<form>
<input/>
<input/>
<textarea ></textarea>
<button>Get in touch</button>
</form>


```